src/nyc_trees/apps/core/templates/core/partials/nav.html

```diff
@@ -0,0 +1,34 @@
+<nav class="nav-side">
+    <div class="nav-side-user clearfix">
+        <div class="nav-side-achievement"></div>
+        <span>
+            <h2><a href="{% url 'user_detail_redirect' %}" class="nav-side-user-name">{{ user.username }}</a></h2>
+            <div class="nav-side-options">
+                <a href="{% url 'user_profile_settings' %}" class="btn btn-primary">Settings</a> <a href="#" class="btn btn-primary">Privacy</a>
+            </div>
+        </span>
+    </div>
+    <ul class="nav-side-list">
+        {% if not soft_launch %}
+        {% if my_events_now|length == 1 %}
+            {% with event=my_events_now.0 %}
+                <li><a href="{{ event.get_user_checkin_url }}">Start Mapping</a></li>
+            {% endwith %}
+        {% endif %}
+        <li{% if active_page == "dashboard" %} class="active"{% endif %}><a href="{% url 'home_page' %}">Dashboard</a></li>
+        <li{% if active_page == "training" %} class="active"{% endif %}><a href="{% url 'training_summary_pure' %}">Training</a></li>
+        <li{% if active_page == "groups" %} class="active"{% endif %}><a href="{% url 'group_list_page' %}">Groups</a></li>
+        <li{% if active_page == "events" %} class="active"{% endif %}><a href="{% url 'events_list_page' %}">Events</a></li>
+        <li{% if active_page == "reservations" %} class="active"{% endif %}><a href="{% url 'reservations' %}">Reservations</a></li>
+        <li{% if active_page == "achievements" %} class="active"{% endif %}><a href="#">Achievements</a></li>
+        {% endif %}
```

Comment:
It's a little hard to determine which `if` this `endif` belongs to.
Consider indenting the block inside the `{% if not soft_launch %}` to make it clearer